iris_doc/ts/api_tagger_ts.py

```diff
@@ -22,13 +22,13 @@ def matchClassConstructor(self, line: str, className: str) -> str:
         return None
 
     def matchMemberFunction(self, line: str) -> str:
+        # TODO: It passes all tests with just the second regex match. Delete first?
```

Comment:
I think this may need @LichKing-2234 to confirm it.

Reply:
I think you can revert this change if this does not affect your logic, @LichKing-2234 may handle it if there is any issue in the future.

Reply:
All I added was my TODO note here, will remove